apps/web/src/components/Renderers/RichTextRenderer/RichTextRender.tsx

```diff
@@ -0,0 +1,63 @@
+import type { Options } from '@contentful/rich-text-react-renderer'
```

Comment:
Can we unit test this?  example of how to do this in [frf](https://github.com/PA-NIHR-CRN/frf-web/blob/779e884d63d463ee1bb8b06b29f126f681ca5e24/src/components/Renderers/RichTextRenderer/RichTextRenderer.spec.tsx)